starfish.js

```diff
@@ -0,0 +1,163 @@
+const {
+    alternateIdColumnNumber,
+    githubIdColumnNumber,
+    githubImportantEvents,
+    githubToken,
+    ignoreSelfOwnedEvents,
+    minimumNumberOfContributions,
+} = require('./globals');
+const { createLuxonDateTimeFromIso } = require('./dateTimes');
+const fetch = require('node-fetch');
+const parse = require('parse-link-header');
+
+function isEventImportant(event) {
+    const type = event.type;
+
+    if (githubImportantEvents.indexOf(type) >= 0) {
+        return true;
+    }
+    if (event.payload) {
+        const typeWithAction = `${type}.${event.payload.action}`;
+        if (githubImportantEvents.indexOf(typeWithAction) >= 0) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+function filterResponseForImportantEvents(allEventsFromFetch) {
+    return allEventsFromFetch.filter((event) => {
+        return isEventImportant(event);
+    });
+}
+
+function shouldIncludeEventBecauseAutherIsNotRepoOwner(eventType) {
```

Comment:
Typo: "Auther".

This function and it's name are really hard for my brain to figure out and keep straight, but I don't have a quick suggestion for a change, so I'm OK leaving it for now.

Reply:
I think I just made this much better. Let me know what you think. We could also keep it as it was, but I like being more explicit with it.